### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by vkaracic**
In this PR I've added two Make commands, one that extract strings that are marked for translation in .py, .js and .html files, and the other that compiles them to .mo files.

TODO:
- [x] Translation documentation
- [ ] Test it live

### CODE REVIEW

**Comment by marjev** [comment]
Just as a reminder:
You'll have to remove `from django.utils.translation import ugettext_lazy as _` and replace it with the XBlock i18n service.

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ python:
   - "2.7"
 
 install:
+  - pip install -U pip wheel
+  - pip install setuptools==35.0.2
```
Comment (vkaracic) [inline .travis.yml:7]: Fix for Travis.

**Comment by vkaracic** [comment]
@marjev ready for re-review.

**Review by marjev — APPROVED** [review]

**Comment by vkaracic** [comment]
Time to internationalize our XBlock! :smile: 
giphy

**Pull request merged**